### PR TITLE
chore(aws): cleanup tests on dynamodb and cloudwatch

### PR DESCRIPTION
**Pull request by sansns**
### Context

Remove search function

### Description

Remove search function from Cloudwatch and DynamoDB tests

### Checklist

- Are there new checks included in this PR? No
    - If so, do we need to update permissions for the provider? Please review this carefully.
- [X] Review if the code is being covered by tests.
- [X] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
- [X] Review if backport is needed.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/providers/aws/services/cloudwatch/cloudwatch_log_group_no_secrets_in_logs/cloudwatch_log_group_no_secrets_in_logs_test.py [hunk 1]
```diff
@@ -11,6 +11,13 @@
     set_mocked_aws_provider,
 )
 
+timestamp = int(unix_time_millis())
+dttimestamp = (
```
Comment (jfagoagas) [inline tests/providers/aws/services/cloudwatch/cloudwatch_log_group_no_secrets_in_logs/cloudwatch_log_group_no_secrets_in_logs_test.py:15]: What is the purpose of this one?

Comment (sansns) [reply tests/providers/aws/services/cloudwatch/cloudwatch_log_group_no_secrets_in_logs/cloudwatch_log_group_no_secrets_in_logs_test.py:15]: timestamp is required to be milliseconds when submitting the put_log_events: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/logs/client/put_log_events.html#put-log-events

dttimestamp is set to format the timestamp to perform the comparison in the results.

Comment (jfagoagas) [reply tests/providers/aws/services/cloudwatch/cloudwatch_log_group_no_secrets_in_logs/cloudwatch_log_group_no_secrets_in_logs_test.py:15]: Thanks for clarifying this!

**Review by jfagoagas — APPROVED** [review]
🔝 🔝 🔝

**Review by jfagoagas — APPROVED** [review]
🔝 🔝 🔝

**Pull request merged**